sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -78,12 +83,28 @@ class DictVectorizer(TransformerMixin, BaseEstimator):
     >>> X
     array([[2., 0., 1.],
            [0., 1., 3.]])
-    >>> v.inverse_transform(X) == \
-        [{'bar': 2.0, 'foo': 1.0}, {'baz': 1.0, 'foo': 3.0}]
+    >>> v.inverse_transform(X) == [{'bar': 2.0, 'foo': 1.0},
+    ...                            {'baz': 1.0, 'foo': 3.0}]
     True
     >>> v.transform({'foo': 4, 'unseen_feature': 3})
     array([[0., 0., 4.]])
 
+    Example with multiple values for one categorical values:
+
+    >>> D2 = [{'foo': '1', 'bar': '2'}, {'foo': '3', 'baz': '1'},
+    ...       {'foo': ['one', 'three']}]
+    >>> X = v.fit_transform(D2)
+    >>> X
+    array([[1., 0., 1., 0., 0., 0.],
+           [0., 1., 0., 1., 0., 0.],
+           [0., 0., 0., 0., 1., 1.]])
+    >>> v.inverse_transform(X) == [{'bar=2': 1.0, 'foo=1': 1.0},
```

Comment:
I don't think we need to demonstrate `inverse_transform` here, but I think `get_feature_names` would be very instructive.